docker-compose.yml

```diff
@@ -35,3 +35,4 @@ services:
     volumes:
       # Sync src folder to allow development without rebuilding on each change
       - ./driver:/app/driver
+      - ./dex-contracts:/app/dex-contracts
```

Comment:
Is this change related to the PR? I'm not sure I understand why it is needed for the instance and result file name changes.

Reply:
It's not required, but more convenient as otherwise you are stuck with the contract definitions from the time you built the container (might be outdated). And rebuilding just to get the contracts in there is time consuming. Will make this a separate PR though.